client/verta/verta/registry/_check_model_dependencies.py

```diff
@@ -23,7 +23,8 @@ def _check_model_dependencies(
     provided environment.
 
     .. note::
-        This function is not guaranteed to detect all dependencies in all cases, and should be considered a fast sanity check rather than a test.
+        This function is not guaranteed to detect all dependencies in all cases,
+        and should be considered a fast sanity check rather than a test.
```

Comment:
The "commit suggestion" button edited this on a previous PR.  This change just adds a newline to make it readable in the file.